server/chat-formatter.ts

```diff
@@ -22,7 +22,7 @@ REGEXFREE SOURCE FOR LINKREGEX
 			\b [a-z0-9-]+ ( \. [a-z0-9-]+ )* \.
```

Comment:
I see there's a word boundary `\b` at the beginning of the domain, so perhaps what this really needs is one at the end?